t/044-req-body.t

```diff
@@ -1744,3 +1744,20 @@ content length: 5
 --- no_error_log
 [error]
 [alert]
+
+
+
+=== TEST 53: HTTP2 read buffered body was discarded
+--- config
+    location = /test {
+        content_by_lua '
+            local err = pcall(ngx.req.read_body())
+            ngx.say(err)
+        ';
```

Comment:
```suggestion
        content_by_lua_block {
            local err = pcall(ngx.req.read_body())
            ngx.say(err)
        }
```

Reply:
you did not fix according to my reivew

Reply:
I chose to commit your suggestion. Does anything I still need to do for this line?

Reply:
My bad, my last commit has overridden some commit.